include/cpr/cprtypes.h

```diff
@@ -12,6 +12,7 @@ struct CaseInsensitiveCompare {
 
 using Header = std::map<std::string, std::string, CaseInsensitiveCompare>;
 using Url = std::string;
+using ProtocolVersion = enum class HTTP { v1x, v2 };
```

Comment:
Since this library is a wrapper on curl, anything a user can set should be allowed here. That means we should include both 1.0 and 1.1, and additionally have a `default` version that sets `CURLOPT_HTTP_VERISON` to `CURL_HTTP_VERSION_NONE`.